### DISCUSSION TITLE
Removing "Select one"

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#15313] [General]
How do I remove the option "Select one" from picklist options when using a form. I would like my option values for the field to be presented without having the option "Select one"

### **Actual Behaviour**
![Screenshot (276)](https://user-images.githubusercontent.com/19745222/137287527-da16e186-29f4-434b-94ae-5a7e82132940.png)

### **Expected Behaviour**
![Screenshot (277)](https://user-images.githubusercontent.com/19745222/137287752-5871e2b6-41c1-4bf9-bcf2-b794a128e38f.png)

**Comment by rskrzypczak** [comment]
**1. Make the field mandatory**
2. Set the default value
